build.gradle.kts

```diff
@@ -13,6 +13,7 @@ plugins {
 
 apply(from = "${project.rootDir}/gradle/githooks.gradle.kts")
 apply(from = "${project.rootDir}/gradle/ktlint.gradle.kts")
+apply(from = "${project.rootDir}/gradle/font.gradle.kts")
```

Comment:
Since it's applied here, every time you sync the project, it will run the gradle task and generate new font files. Is this expected?

Reply:
Not intended. Thank you for noticing. Will be removed.